edi_exchange_template_oca_json/models/edi_exchange_template_output.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2025 Camptocamp SA
+# @author Italo LOPES <italo.lopes@camptocamp.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo import fields, models
+
+
+class EDIExchangeOutputTemplate(models.Model):
+    _inherit = "edi.exchange.template.output"
+
+    generator = fields.Selection(
+        selection_add=[("json", "JSON")],
+        ondelete={"json": "set default"},
+    )
+
+    def _generate_json(self, exchange_record, **kw):
+        """Generate JSON output."""
+        result = self._render_json_values(exchange_record, **kw)
+        return result
+
+    def _post_process_output(self, output):
+        result = super()._post_process_output(output)
+        if self.generator == "json":
+            result = self._post_process_json_output(result)
+        return result
+
+    def _render_json_values(self, exchange_record, **kw):
```

Comment:
this module shouldn't rely on `jsonifier`.
You should be able to generate JSOn as you prefer.

The snippet can simply be something like
```
exporter = env.ref(...)
parser = exporter.get_json_parser()
result = record.jsonifier(parser, [one=True])
```
The role of this module is only to handle and validate the JSON result.